Shoko.Server/Services/VideoLocal_PlaceService.cs

```diff
@@ -410,9 +410,10 @@ public MoveFileResult MoveFile(SVR_VideoLocal_Place videoLocalPlace, bool delete
 
             if (dropFolder.IsDropSource == 1 && deleteEmpty)
             {
-                var directories = dropFolder.BaseDirectory.EnumerateDirectories("*", new EnumerationOptions() { RecurseSubdirectories = true, ReturnSpecialDirectories = true, IgnoreInaccessible = true })
+                var directories = dropFolder.BaseDirectory.EnumerateDirectories("*", new EnumerationOptions() { RecurseSubdirectories = true, ReturnSpecialDirectories = false, IgnoreInaccessible = true })
```

Comment:
Just remove the option if it's false by default. (Do check please, since i can't right now.)

Reply:
The docs do not mention anything but a quick run without the option confirmed its false by default.